### PR TITLE
Topics: add Lightning Gossip

### PR DESCRIPTION
**Pull request by harding**
I thought we were going to have a v2 gossip PR for newsletter 308, so I wrote this topic page, but it turned out to be v2 of a custom LDK protocol.  Oh, well, it's a good topic to have anyway!

### CODE REVIEW

File: _topics/en/lightning-gossip.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  linking node and channel activity to a specific UTXO.
+
+A proposed upgrade to LN gossip, sometimes called "v1.5 gossip", would
+allow using [P2TR][topic taproot] outputs with keypath-signed channel
+announcements and updates, addressing the problem of restricted
+upgrades.
+
+A more ambitious proposed upgrade to LN gossip, sometimes called "v2
+gossip", proposes to address both linkability and restricted upgrades by
+allowing signatures for a broad range of UTXOs to used---not just UTXOs
```
Comment (LarryRuane) [inline _topics/en/lightning-gossip.md:151]:
```suggestion
allowing signatures for a broad range of UTXOs to be used---not just UTXOs
```

File: _topics/en/lightning-gossip.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+that match an LN template.  This would allowing breaking the connection
```
Comment (LarryRuane) [inline _topics/en/lightning-gossip.md:152]:
```suggestion
that match an LN template.  This would allow breaking the connection
```

**Review by murchandamus — COMMENTED** [review]
Looks good to me except the two comments already made by Larry.

File: _topics/en/lightning-gossip.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+## Optional.  Force the display (true) or non-display (false) of stub
+## topic notice.  Default is to display if the page.content is below a
+## threshold word count
+#stub: false
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Lightning gossip** is the decentralized protocol used announce the
```
Comment (bitschmidty) [inline _topics/en/lightning-gossip.md:110]:
```suggestion
  **Lightning gossip** is the decentralized protocol used to announce the
```

File: _topics/en/lightning-gossip.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+---
+Many LN nodes choose to announce their existence and their channels to
+encourage other nodes to use them for routing payments.  Other nodes
+choose to remain private, using [unannounced channels][topic unannounced
+channels].
+
+As of this writing, many developers call the currently deployed
+[BOLT7][] gossip protocol "gossip v1".  One key feature of gossip v1
+is that channel announcements and updates must be signed by a key
```
Comment (bitschmidty) [inline _topics/en/lightning-gossip.md:121]: Maybe make a channel announcements topic reference

File: _topics/en/lightning-gossip.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+  - title: LN developer meeting discussion about v2 gossip improvements
+    url: /en/newsletters/2022/06/15/#gossip-network-updates
+
+  - title: "Core Lightning #5239 improves its gossip handling code"
+    url: /en/newsletters/2022/07/13/#core-lightning-5239
+
+  - title: LN fee rate cards proposed to reduce fee-related channel update gossip
+    url: /en/newsletters/2022/09/28/#ln-fee-ratecards
+
+  - title: LDK #2198 increases the amount before gossiping that a channel is down
```
Comment (bitschmidty) [inline _topics/en/lightning-gossip.md:82]:
```suggestion
  - title: "LDK #2198 increases the amount before gossiping that a channel is down"
```

File: _topics/en/lightning-gossip.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+    url: /en/newsletters/2023/04/26/#ldk-2198
+
+  - title: LDK #2222 allows accepting gossip without verifying it first
```
Comment (bitschmidty) [inline _topics/en/lightning-gossip.md:85]:
```suggestion
  - title: "LDK #2222 allows accepting gossip without verifying it first"
```

File: _topics/en/lightning-gossip.md (hunk 4 above)
Comment (harding) [reply _topics/en/lightning-gossip.md:121]: Oh, geeze, that topic and this topic are really the same.  :man_facepalming:  I should just move this content into there.  Thanks for reminding me that existed!

**Comment by harding** [comment]
Closing for being a duplicate topic. https://github.com/bitcoinops/bitcoinops.github.io/pull/1736#discussion_r1716928022

**Pull request closed without merging**